Python/Filters/tdms_to_h5.py

```diff
@@ -0,0 +1,275 @@
+    return self.laser_offset
+
+  def _set_input_folder(self, value: str) -> None:
+    self.input_folder = value
+
+  def _get_input_folder(self) -> str:
+    return self.input_folder
+
+  def _set_output_folder(self, value: str) -> None:
+    self.output_folder = value
+
+  def _get_output_folder(self) -> str:
+    return self.output_folder
+
+  def _set_prefix(self, value: str) -> None:
+    self.prefix = value
+
+  def _get_prefix(self) -> str:
+    return self.prefix
+
+  def _set_group(self, value: str) -> None:
+    self.group = value
+
+  def _get_group(self) -> str:
+    return self.group
+
+  @staticmethod
+  def name() -> str:
+    return 'TDMS to H5'
+
+  @staticmethod
+  def uuid() -> str:
+    return '{8b069c55-6d94-4db1-b012-cdfb7dd08ad6}'
+
+  @staticmethod
+  def group_name() -> str:
+    return 'Example'
+
+  @staticmethod
+  def sub_group_name() -> str:
+    return 'Sub Example'
+
+  @staticmethod
+  def human_label() -> str:
+    return 'Convert TDMS to HDF5'
+
+  @staticmethod
+  def version() -> str:
+    return '1.0.0'
+
+  @staticmethod
+  def compiled_lib_name() -> str:
+    return 'Python'
+
+  def setup_parameters(self) -> List[FilterParameter]:
+    req = MultiDataArraySelectionFilterParameter.RequirementType([IGeometry.Type.Image], [AttributeMatrix.Type.Cell], [], [])
+    return [
+      IntFilterParameter('Area Offset', 'area_offset', self.area_offset, FilterParameter.Category.Parameter, self._set_area_offset, self._get_area_offset, -1),
+      IntFilterParameter('Intensity Offset', 'intensity_offset', self.intensity_offset, FilterParameter.Category.Parameter, self._set_intensity_offset, self._get_intensity_offset, -1),
+      IntFilterParameter('Laser Offset', 'laser_offset', self.laser_offset, FilterParameter.Category.Parameter, self._set_laser_offset, self._get_laser_offset, -1),
+      InputPathFilterParameter('Input Folder', 'Input Folder', '', FilterParameter.Category.Parameter,
+                               self._set_input_folder, self._get_input_folder, -1),
+      OutputPathFilterParameter('Output Folder', 'Output Folder', '', FilterParameter.Category.Parameter,
+                               self._set_output_folder, self._get_output_folder, -1),
+      StringFilterParameter('Prefix', 'prefix', self.prefix, FilterParameter.Category.Parameter, self._set_prefix, self._get_prefix, -1),
+      StringFilterParameter('Group', 'group', self.group, FilterParameter.Category.Parameter, self._set_group,
+                           self._get_group, -1)
+    ]
+
+  def data_check(self, dca: DataContainerArray, delegate: Union[FilterDelegateCpp, FilterDelegatePy] = FilterDelegatePy()) -> Tuple[int, str]:
+    if not self.input_folder:
+      return (-5550, 'An input folder must be selected')
+
+    if not self.output_folder:
+      return (-301, 'An output folder must be selected')
+
+    if not os.path.exists(self.input_folder):
+      return (-302, 'Input path does not exist')

```

Comment:
When reporting error messages it is always nice to feedback to the user exactly what path does not exist.  As part of the error string also include the path that is reporting the error. That way the end user can double check that they path was entered correctly.